man/docker-images.1.md

```diff
@@ -103,6 +103,24 @@ which displayed different visualizations of the image data. Docker core removed
 this functionality in the 1.7 version. If you liked this functionality, you can
 still find it in the third-party dockviz tool: https://github.com/justone/dockviz.
 
+## Listing images in a desired format
+
+You can use the `--format` parameter to list only the image attributes required
+and in the order required. For example, if the image ID, Tag, and REPOSITORY 
+are required, in that order, then the following command is used:
+
+    docker images --format "{{.ID}}: {{.Tag}}\t{{.Repository}}"
+
+It produces output similar to :
+
+    11a5207645d4: latest	docker.io/fedora
+    4211be3df429: latest	docker.io/ubuntu
+    f7c419c1256d: latest	ipbabble/fedora-httpd
+    ce859f6e9f3d: latest	fedora-sshd
+    f9d5dee79539: latest	docker.io/kubernetes
+
+Valid template placeholders are listed above.
```

Comment:
If you're interested in working on this, you can update this PR by making those changes, and "amending" your commit (`git add man/docker-images.1.md`, `git commit -s --amend --no-edit`, then force push; `git push -f origin`). That should update the PR here on GitHub